apps/api-gateway/src/api/benchmarks.py

```
@@ -228,3 +228,79 @@ async def cross_store_insights(
+import os
+from datetime import date, timedelta
+
+from src.core.dependencies import get_current_user
+from src.lib.benchmark import BenchmarkService
+
+legacy_benchmark_router = APIRouter(prefix="/benchmark", tags=["benchmark"])
+
+
+@legacy_benchmark_router.get("/report")
+async def get_benchmark_report(
+    start_date: date = Query(..., description="开始日期"),
```

Ruff (0.15.13)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)